collector/src/main/java/com/bitmovin/analytics/BitmovinAnalyticsConfig.java

```diff
@@ -1,44 +1,118 @@
 package com.bitmovin.analytics;
 
 import android.content.Context;
+import android.os.Parcel;
+import android.os.Parcelable;
 
 import com.bitmovin.analytics.enums.CDNProvider;
 import com.bitmovin.analytics.enums.PlayerType;
 
-public class BitmovinAnalyticsConfig {
-    public static final String analyticsUrl = "https://analytics-ingress-global.bitmovin.com/analytics";
-    public static final String licenseUrl = "https://analytics-ingress-global.bitmovin.com/licensing";
-
-    private String key;
-    private String playerKey;
+public class BitmovinAnalyticsConfig implements Parcelable {
+    private String analyticsUrl = "https://analytics-ingress-global.bitmovin.com/analytics";
     private String cdnProvider;
-    private String videoId;
-    private PlayerType playerType;
-    private String customUserId;
     private String customData1;
     private String customData2;
     private String customData3;
     private String customData4;
     private String customData5;
-    private String title;
+    private String customUserId;
     private String experimentName;
+    private int heartbeatInterval = 59700;
+    private String licenseUrl = "https://analytics-ingress-global.bitmovin.com/licensing";
+    private String key;
+    private String title;
     private String path;
+    private String playerKey;
+    private PlayerType playerType;
+    private String videoId;
     private Context context;
-    private int heartbeatInterval = 59700;
 
+    public static final Creator<BitmovinAnalyticsConfig> CREATOR = new Creator<BitmovinAnalyticsConfig>() {
```

Comment:
What does this do? 
I don't see it being used anywhere

Reply:
It is used by Android  (e.g. see https://medium.com/@nikhildhyani365/understand-parcelable-in-android-27ce420d695b, https://www.vogella.com/tutorials/AndroidParcelable/article.html)